src/policy/CMSRucioPolicy/__init__.py

```diff
@@ -0,0 +1,18 @@
+# CMSPolicyPackage
+#
+# Eric Vaandering <ewv@fnal.gov>, 2022
+
+from CMSRucioPolicy.algorithms import lfn2pfn
+
+SUPPORTED_VERSION = ["1.30"]
```

Comment:
I'm going to put this into 1.31, not 1.30.

Reply:
Added 1.31 too.